scripts/create_test

```diff
@@ -257,7 +257,7 @@ OR
         args.no_batch = True
 
     if args.test_id is None:
-        args.test_id = CIME.utils.get_timestamp()
+        args.test_id = "%s.%s"%(CIME.utils.get_timestamp(), random.randrange(0,99,1))
```

Comment:
Yeah. I'm thinking that maybe the hex characters might be more obvious to users (makes it clear it's not a date), but I don't particularly care.